src/lib/storage/src/benches/databases.rs

```diff
@@ -71,7 +71,8 @@ pub fn database_benchmarks(c: &mut Criterion) {
             temps.push(db_file);
             backend
         });
-
+        
```

Comment:
All of this could just be in one, feature-gated block.
```rs
#[cfg(feature = "rocksdb")]
{
    let mut rocksdb_backend = ...;
    // Everything else
}